pkg/mysql/getters.go

```diff
@@ -0,0 +1,36 @@
+package mysql
+
+import (
+	"context"
+	apiv1alpha1 "github.com/percona/percona-server-mysql-operator/api/v1alpha1"
+	"github.com/percona/percona-server-mysql-operator/pkg/k8s"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetReadyMySQLPod(ctx context.Context, cl client.Reader, cr *apiv1alpha1.PerconaServerMySQL) (*corev1.Pod, error) {
```

Comment:
we can name this GetReadyPod since it will be used as mysql.GetReadyPod, mysql in the name of function is redundant